pupil_src/shared_modules/gaze_producers.py

```diff
@@ -562,9 +562,15 @@ def toggle_marker_detection(self):
             self.start_marker_detection()
 
     def start_marker_detection(self):
-        self.process_pipe = zmq_tools.Msg_Pair_Server(self.g_pool.zmq_ctx)
         self.circle_marker_positions = []
-        source_path = self.g_pool.capture.source_path
+
+        try:
+            source_path = self.g_pool.capture.source_path
```

Comment:
why? Fake capture is api conform. self.g_pool.capture.source_path should point to `dfn/dfdfgs/world.fake`. We should still be able to run detection on it.